website/blog/2023-02-01-technical-preview/index.mdx

```diff
@@ -0,0 +1,113 @@
+
+
+## The missing package manager for Kubernetes - Introducing Glasskube
+
+Package Management on Kubernetes is one of the most pressing issues in the Cloud Native community.
+A concept which is widely known from other ecosystems like desktop and mobile computing has yet not been realized for cloud computing.
+On Android and iOS for example millions of developers publish their packages via the Play store or App Store to its users.
+The package manager also makes sure all users receive the latest version published by the developer and the developer receives
+crash reports and user feedback as a return to improve their applications,
+but as a cloud native developer there is no package manager you can rely on - yet.
+
+Our first release (v0.0.1) already features a working prototype that can install basic packages, but a lot of challenges still need to be solved.
+
+### A cloud native architecture
+
+Glasskube is designed as a cloud native application itself.
+Featuring an easy to install client that comes with a graphical user interface and autocompletion for your favorite shell.
+
+At the heart of the Glasskube package ecosystems lies our central package registry which holds the package manifests.
+The Glasskube package operator syncs the latest manifest into the cluster and makes sure it will be updated as soon as a new manifest is available.
+
+```mermaid
+---
+title: glasskube install [package]
+---
+flowchart BT
+  UI([UI])-- via local server<br>http://localhost:8580 ---Client(Client)
+  CLI([CLI])-- cobra cli ---Client
+  Client-- 1. validate package -->Repo[(Public Glasskube<br>Package Repo)]
+  Client-- 2. create<br>`Package` CR -->Kubernetes(((Kubernetes API)))
+  subgraph Cluster
+    Kubernetes-- 3. reconcile<br>`Package` -->PackageController
+    PackageController-- 4. create `PackageInfo`<br>if not present-->Kubernetes
+    Kubernetes-- 5. reconcile<br>`PackageInfo`-->PackageInfoController
+    end
+  PackageInfoController<-- 6. update package manifest -->Repo
+  subgraph Cluster
+    PackageInfoController-- 7. update manifest<br>in `PackageInfo` -->Kubernetes
+    Kubernetes-- 8. reconcile<br>`PackageInfo` -->PackageController
+    PackageController-- 9. deploy package -->Kubernetes
+  end
+
+  Kubernetes-- 10. package status -->Client
+```
+
+### Challenges that need to be solved
+
+We already covered some our next features in our [public roadmap](https://glasskube.dev/roadmap), but I also like to shortly speak about broader challenges.
+
+#### Kubernetes version compatibility
+
+Kubernetes releases minor versions every 4 months which often comes with new api versions. So package authors need to adapt their packages to the new api versions.
+Kubernetes is designed in a way that often multiple api versions are implemented in specific release of Kubernetes, so packages often work with a broader range
```

Comment:
```suggestion
In Kubernetes, a particular release might include more than one API version of a resource, so that packages can be compatible with a broader range
```